docs/lib/predictions/fragments/ios/getting-started.md

```diff
@@ -109,8 +109,12 @@ Add the dependencies to the `Podfile`:
 ```ruby
 target :'YOUR-APP-NAME' do
 	use_frameworks!
-	pod 'AWSPredictionsPlugin'
-	pod 'AWSMobileClient'
+    pod 'Amplify'
+    pod 'AmplifyPlugins'
+    pod 'AWSPluginsCore'
+    pod 'CoreMLPredictionsPlugin'
+    pod 'AWSPredictionsPlugin'
+    pod 'AWSMobileClient'
```

Comment:
i think it should be removed since Amplify doesn't have a version, it will pick up the latest, which then could depend on a AWSMobileClient's version specified in the podspec.  Putting a version means we have to maintain this. there has been some conversation around taking the version out from all places. this appears to be the first place that we're doing it. 

comes to think of it, we don't even need to specify AWSMobileClient in here...
```
pod 'CoreMLPredictionsPlugin'
pod 'AWSPredictionsPlugin'
```

pulls in 
```
Installing AWSAuthCore (2.12.7)
Installing AWSCognitoIdentityProvider (2.12.7)
Installing AWSCognitoIdentityProviderASF (1.0.1)
Installing AWSComprehend (2.12.7)
Installing AWSCore (2.12.7)
Installing AWSMobileClient (2.12.7)
Installing AWSPluginsCore (0.10.0)
Installing AWSPolly (2.12.7)
Installing AWSPredictionsPlugin (0.10.0)
Installing AWSRekognition (2.12.7)
Installing AWSTextract (2.12.7)
Installing AWSTranscribe (2.12.7)
Installing AWSTranslate (2.12.7)
Installing Amplify (0.10.0)
Installing CoreMLPredictionsPlugin (0.10.0)
```
note until we release amplify-ios to depend on 2.13.x, the current version (0.10.0 depends on 2.12.x )

Reply:
removed from master branch here: https://github.com/aws-amplify/docs/pull/1452

Reply:
@nikhil-dabhade  could you remove everything except for 
```
pod 'CoreMLPredictionsPlugin'
pod 'AWSPredictionsPlugin'
``` 
from this v2-preview PR?

Reply:
Yes. Updating the PR to include this.

Reply:
@jpignata For the page ios/manualsetup.md it is picked up by your PR https://github.com/aws-amplify/docs/pull/1453/files so that should be covered.

Reply:
@jpignata For #1226  removing the version is the way we are going on this one and it is already ported and tested.
For #1112, they are already ported 
https://docs.amplify.aws/sdk/auth/custom-auth-flow?platform=ios#lambda-trigger-setup
https://docs.amplify.aws/lib/auth/custom-auth-flow?platform=ios#lambda-trigger-setup